libraries/tools/kotlin-gradle-plugin-integration-tests/src/test/kotlin/org/jetbrains/kotlin/gradle/AbstractKotlinAndroidGradleTests.kt

```diff
@@ -427,6 +427,21 @@ open class KotlinAndroid36GradleIT : KotlinAndroid34GradleIT() {
             assertSuccessful()
         }
     }
+
+    @Test
+    fun testJvmWithJavaConfigurationCache() =
+        with(Project("mppJvmWithJava", gradleVersionRequirement = GradleVersionRequired.AtLeast("7.1"))) {
```

Comment:
This test is not related to Android support, so putting it in `KotlinAndroid36GradleIT` doesn't look right, especially since this test will run more than once: here and in the subtype `KotlinAndroid70GradleIT` with a different AGP version.

Reply:
@gavra0 Could you please move it to some other test class, such as `org.jetbrains.kotlin.gradle.ConfigurationCacheIT`?

Reply:
The test is now in `ConfigurationCacheIT`.